flexmeasures/data/tests/test_queries.py

```diff
@@ -225,6 +225,8 @@ def test_query_beliefs(setup_beliefs):
     source = DataSource.query.filter_by(name="Seita").one_or_none()
     bdfs = [
         TimedBelief.search(sensor, source=source),
+        TimedBelief.search(sensor.id, source=source),
+        TimedBelief.collect(sensor.name, source=source),
         sensor.search_beliefs(source=source),
         tb.BeliefsDataFrame(sensor.beliefs),  # doesn't allow filtering
```

Comment:
So are all beliefs in `setup_beliefs` from one source anyways? Then we are not really tresting the filtering.

Reply:
True. We're using the filter only to test whether a FlexMeasures `DataSource`, which subclasses the timely-beliefs `BeliefSource`, results in a query that the database can still understand.